### PR TITLE
created makeStyles() example, and added template

### PR DESCRIPTION
**Pull request by rymaju**
- Added a CSS-in-JS example on ExamplePage
- Added a Template.jsx file as a quickstart for future page components

### CODE REVIEW

**Review by jackblanc — APPROVED** [review]

**Comment by rymaju** [comment]
image

File: src/App.tsx [hunk 1]
```diff
@@ -6,31 +6,68 @@ import Home from './components/Home';
...
+      // dark: will be calculated from palette.primary.main,
+      // contrastText: will be calculated to contrast with palette.primary.main
+    },
+    text: {
+      //primary: '',
+      //secondary: ''
+    },
+  },
+  typography: {
+    h1: {
```
Comment (jackblanc) [inline src/App.tsx]: should this be applied to p tags too? We use it for everything, including google docs etc

File: src/App.tsx [hunk 2]
```diff
@@ -6,31 +6,68 @@ import Home from './components/Home';
 import ExamplePage from './components/ExamplePage';
 import NotFound from './components/NotFound';
 
-import { AppBar, Tabs, Tab } from '@material-ui/core';
+import { AppBar, Tabs, Tab, ThemeProvider } from '@material-ui/core';
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const theme = createMuiTheme({
```
Comment (jackblanc) [inline src/App.tsx]: should/can theme go in it's own file?

File: src/App.tsx [hunk 3]
```diff
@@ -6,31 +6,68 @@ import Home from './components/Home';
...
-          </Tabs>
-        </AppBar>
-        <Switch>
-          <Route path="/" exact component={Home} />
-          <Route path="/example" exact component={ExamplePage} />
-          <Route path="*" exact component={NotFound} />
-        </Switch>
-      </Router>
+        <Router>
+          <AppBar position="static" color="transparent">
```
Comment (jackblanc) [inline src/App.tsx]: Can AppBar be its own component? There will ultimately be some decent functionality here

**Review by jackblanc — COMMENTED** [review]
This is great! Left a few suggestions

**Review by jackblanc — APPROVED** [review]
Awesome work

**Pull request merged**